service/gcs/run_unittests

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex 
```

Comment:
I cannot think of a scenario that someone wants to setup the test environment without wanting to run those unittests.